### PR TITLE
[dom-gpu] QuantumESPRESSO with perftools-lite/6.5.1

### PR DESCRIPTION
**Pull request by lucamar**
Unset the following variable defined by `perftools-lite/6.5.1` is needed during configuration: `CRAYPAT_COMPILER_OPTIONS`

The build would fail otherwise with the following error on the login nodes only:
```
checking whether the Fortran compiler works... configure: error: in `/dev/shm/lucamar/easybuild/stage/build/QuantumESPRESSO/6.1.0/CrayIntel-17.08-pat-6.5.1/qe-6.1':
configure: error: cannot run Fortran compiled programs.
If you meant to cross compile, use `--host'.
```

An alternative option might be using `--host`, as suggested by the error message itself.
The variable is then exported with the value `1` before building.

The previous recipe was instead successful on the compute nodes.

### CODE REVIEW

**Comment by lucamar** [comment]
retest this please

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.1.0-CrayIntel-17.08-pat-6.5.1.eb [hunk 1]
```diff
@@ -23,13 +23,14 @@ builddependencies = [
     ('perftools-lite/%s' % patversion, EXTERNAL_MODULE)
 ]
 
-preconfigopts = ' CC=cc LDFLAGS+="$LDFLAGS -qopenmp" && ' 
+preconfigopts = ' unset CRAYPAT_COMPILER_OPTIONS && '
```
Comment (jgphpc) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.1.0-CrayIntel-17.08-pat-6.5.1.eb]: You could also try with:

```
preconfigopts = 'unset CRAYPAT_LITE &&'                                         
buildopts = ' CRAYPAT_LITE=sample_profile '               
```

Comment (lucamar) [reply easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.1.0-CrayIntel-17.08-pat-6.5.1.eb]: If I set `CRAYPAT_LITE=sample_profile` when building, can I change profile later at runtime?

**Comment by lucamar** [comment]
Thanks @jgphpc, I have update the EasyBuild configuration file according to your advice. I don't know why the download failed yesterday, I have tried on the compute nodes today the same link and it worked.

**Pull request merged**